src/arch/host/include/arch/lib/cache.h

```diff
@@ -10,14 +10,859 @@
 #ifndef __ARCH_LIB_CACHE_H__
 #define __ARCH_LIB_CACHE_H__
 
+#define CACHE_VCORE_COUNT	4
```

Comment:
should we not just use the core count from Kconfig?

Reply:
fixed